js/proposalGenerator.js

```diff
@@ -63,6 +63,12 @@ ProposalGenerator.prototype.validate = function() {
                 $('#payment_address').val("Invalid payment address.");
                 break;
 
+            case 'Invalid P2SHAddress':

```

Comment:
Where is this message generated? I didn't see any changes for it, so wondering where that comes from.

Reply:
From bitcore-lib-dash:
here: https://github.com/dashpay/bitcore-lib-dash/pull/21/commits/cd933b701401edab46ac91dcf9fc71dd9bb04345
and here: https://github.com/dashpay/bitcore-lib-dash/pull/21/commits/6563206dfa2a116dffc9749d0e4b5fe23f3e7ed0

Reply:
Ah, ok... so we have specific logic for governance proposals built into `bitcore-lib-dash`?

Reply:
I believe we'll need a new version of bitcore-lib-dash referenced in the `package.json` for this to take effect, right @Alex-Werner ?

Also, we should sort out which of the repositories (dashevo/dashpay) versions we'll use and remove the other. I think we'd settled on dashevo, but we should re-visit and get that sorted.

Reply:
Exact, we will need to release a new version of BLD : https://github.com/dashevo/bitcore-lib-dash/pull/3 
We always use Dashevo, we indeed should redirect dashpay repo to dashevo one.